hack/dockerfiles/generated-files.Dockerfile

```diff
@@ -8,7 +8,12 @@ ARG GOGO_VERSION=v1.3.2
 RUN apt-get update && apt-get --no-install-recommends install -y \
 	unzip \
 	&& true
-RUN wget -q https://github.com/google/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip && unzip protoc-${PROTOC_VERSION}-linux-x86_64.zip -d /usr/local
+
+ARG TARGETOS TARGETARCH
+RUN set -e; \
+	if [ "$TARGETARCH" = "amd64" ]; then TARGETARCH=x86_64; fi; \
+	if [ "$TARGETARCH" = "arm64" ]; then TARGETARCH=aarch_64; fi; \
```

Comment:
Can we avoid overwriting `TARGETARCH`?

e.g. `arch=$(echo $TARGETARCH | sed -e s/amd64/x86_64/ -e s/arm64/aarch_64); wget ...${arch}...`